samples/create-training-pipeline-tabular-classification.js

```diff
@@ -24,7 +24,7 @@ async function main(
   project,
   location = 'us-central1'
 ) {
-  // [START aiplatform_create_training_pipeline_tables_classification]
+  // [START aiplatform_create_training_pipeline_tables_classification_sample]
```

Comment:
change this from "tables" to "tabular".

Reply:
this does not seem to have been updated to "tabular" (and the other instances below)